mappings/net/minecraft/block/Block.mapping

```diff
@@ -1,11 +1,11 @@
 CLASS none/akw net/minecraft/block/Block
 	FIELD A stateFactory Lnone/ask;
-	FIELD a id Lnone/kp;
+	FIELD a ID Lnone/kp;
 	FIELD b creativeTab Lnone/adq;
 	FIELD c defaultState Lnone/asj;
 	FIELD d unlocalizedName Ljava/lang/String;
-	FIELD h registry Lnone/cs;
-	FIELD i blockstateList Lnone/cx;
+	FIELD h REGISTRY Lnone/cs;
+	FIELD i BLOCKSTATELIST Lnone/cx;
```

Comment:
Should be `BLOCKSTATE_LIST` then. Not sure if all static final things should be written like that or only fields that are immutable.